src/coreclr/vm/clsload.cpp

```diff
@@ -3865,11 +3587,13 @@ VOID ClassLoader::AddExportedTypeDontHaveLock(Module *pManifestModule,
     AddExportedTypeHaveLock(
         pManifestModule,
         cl,
+        NULL,
```

Comment:
Remove `NULL` and use empty `SArray<EEClassHashEntry_t *>`?

Reply:
Sure. I did it for the other case.